specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -1473,6 +1606,32 @@
       "required": true,
       "type": "string",
       "x-ms-parameter-location": "method"
+    },
+    "externalCloudProviderTypeParameter": {
+      "name": "externalCloudProviderType",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "externalSubscriptions",
+        "externalBillingAccounts"
+      ],
+      "x-ms-enum": {
+        "name": "externalCloudProviderType",
+        "modelAsString": true
+      },
+      "description": "The external cloud provider type associated with dimension/query operations. This includes 'externalSubscriptions' for linked account and 'externalBillingAccounts' for consolidated account.",
+      "x-ms-parameter-location": "method",
+      "x-ms-skip-url-encoding": true
```

Comment:
You shouldn't need to set `x-ms-skip-url-encoding` here

Reply:
Sure, I removed it. thank you for the quick review